impl/pkg/dht/dht_test.go

```diff
@@ -13,8 +14,19 @@ import (
 
 	"github.com/TBD54566975/did-dht-method/internal/util"
 	dhtclient "github.com/TBD54566975/did-dht-method/pkg/dht"
+	"github.com/TBD54566975/did-dht-method/pkg/telemetry"
 )
 
+func TestMain(m *testing.M) {
+	// telemetry.Tracer will be nil if this isn't called before the test
+	if err := telemetry.SetupTelemetry(context.Background()); err != nil {
+		panic(err)
```

Comment:
is there a `m.Fail()` equivalent so we don't need to panic?

also should the contexts here and below be the same?